<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnOn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>700</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>On/Off</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUp">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>550</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRight">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>600</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLeft">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>600</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDown">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>640</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelector">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>600</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start/Stop</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>481</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="page_Off"/>
    <widget class="QWidget" name="pageMenu">
     <widget class="QPushButton" name="btnSettings">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>50</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLog">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>90</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Log</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnNewSession">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>130</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Start New Session</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageSettings">
     <widget class="QPushButton" name="btnChallenge">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>34</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Change Challenge Level</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnGoToBreath">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Change Breath Pacer Level</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSetMenu">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>200</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Menu</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageChallenge">
     <widget class="QRadioButton" name="btnL1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>96</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="btnL2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>96</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="btnL3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>96</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="btnL4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>96</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblCHallege">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Change Challenge Level</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDone">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Done</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMenu">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>170</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Menu</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageBreath">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>221</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose a number between 1-30</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBreath">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Done</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtBreath">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBreathMenu">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>90</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Menu</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageSessionChart">
     <widget class="QGroupBox" name="chartGroupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>491</width>
        <height>371</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QChartView" name="graphicsView">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>70</y>
         <width>481</width>
         <height>301</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="readingLED">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>20</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Coherence</string>
       </property>
      </widget>
      <widget class="QLabel" name="cohChartLabel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>[ ]</string>
       </property>
      </widget>
      <widget class="QLabel" name="lenChartLabel">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Length (s)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>20</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Achievement</string>
       </property>
      </widget>
      <widget class="QLabel" name="achChartLabel">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>40</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>[ ]</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="pageSummary">
     <widget class="QGroupBox" name="summary">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>0</y>
        <width>391</width>
        <height>371</height>
       </rect>
      </property>
      <property name="title">
       <string>Summary</string>
      </property>
      <widget class="QLabel" name="cLevel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Challenge Level</string>
       </property>
      </widget>
      <widget class="QLabel" name="pTimeCoh">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>% Time in Coherence</string>
       </property>
      </widget>
      <widget class="QLabel" name="greenCoh">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>41</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Green</string>
       </property>
      </widget>
      <widget class="QLabel" name="blueCoh">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>100</y>
         <width>41</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Blue</string>
       </property>
      </widget>
      <widget class="QLabel" name="redCoh">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>130</y>
         <width>41</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Red</string>
       </property>
      </widget>
      <widget class="QLabel" name="avgCoh">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>160</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Average Coherence</string>
       </property>
      </widget>
      <widget class="QLabel" name="sessionLen">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>190</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Length of Session</string>
       </property>
      </widget>
      <widget class="QLabel" name="achieveScore">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>220</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Achievement Score</string>
       </property>
      </widget>
      <widget class="QLabel" name="hrvGraph">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>250</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>HRV Graph</string>
       </property>
      </widget>
      <widget class="QLabel" name="cLevelVal">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>54</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="greenCohVal">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>54</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="blueCohVal">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>100</y>
         <width>54</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="redCohVal">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>130</y>
         <width>54</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="avgCohVal">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>160</y>
         <width>54</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="sessionLenVal">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>190</y>
         <width>54</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="achieveScoreVal">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>220</y>
         <width>54</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="date">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>30</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
      <widget class="QLabel" name="dateVal">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>50</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSummaryMenu">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>340</y>
         <width>80</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Menu</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnBack">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>340</y>
         <width>80</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="pageHistory">
     <widget class="QGroupBox" name="history2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>391</width>
        <height>361</height>
       </rect>
      </property>
      <property name="title">
       <string>History</string>
      </property>
      <widget class="QListWidget" name="summaryList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>371</width>
         <height>261</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnHistoryMenu">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>330</y>
         <width>80</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Menu</string>
       </property>
      </widget>
      <widget class="QPushButton" name="view">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>300</y>
         <width>91</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>View Summary</string>
       </property>
      </widget>
      <widget class="QPushButton" name="reset">
       <property name="geometry">
        <rect>
         <x>301</x>
         <y>300</y>
         <width>80</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
      <widget class="QPushButton" name="remove">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>300</y>
         <width>101</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete Summary</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="pageLed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>561</width>
      <height>51</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="led_red">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="led_blue">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="led_green">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>510</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="createSummary">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Summary</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>665</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
